tests/server_tests.py

```diff
@@ -159,6 +160,13 @@ def __init__(self, port, smtp_port):
             '--datastore_consistency_policy=consistent',
         ])
 
+    def flush_output(self):
+      """Flushes the buffered output from this subprocess to stderr."""
+      self.output, lines_to_print = [], self.output
```

Comment:
Nit pick, but lines_to_print is not a good name in this context because we won't print these lines. Maybe "original_output"?